modules/ossm-deploying-automatic-gateway-injection.adoc

```diff
@@ -74,8 +86,69 @@ subjects:
 - kind: ServiceAccount
   name: default
 ----
-<1> Select the gateway injection template rather than the default sidecar template.
-<2> Set a unique label for the gateway. This setting is required to ensure Gateways can select this workload.
-<3> Enable gateway injection. If connecting to a revisioned control plane, replace with `istio.io/rev: revision-name`.
-<4> The image automatically updates each time the pod starts.
-<5> Set up roles to allow reading credentials for TLS.
+
+. Grant access to the new gateway from outside the cluster. This is required whenever `spec.security.manageNetworkPolicy` is set to `true`.
```

Comment:
```suggestion
. Grant access to the new gateway from outside the cluster, which is required whenever `spec.security.manageNetworkPolicy` is set to `true`.
```
- Not sure if this is the best way to solve it, but I always try to avoid the word "this" by itself. Imo, it should always be "this xyx" because the antecedent is not clear most of the time